UnityProject/Assets/Scripts/Drawers/Drawer.cs

```diff
@@ -211,7 +213,8 @@ public virtual void CloseDrawer()
 		GatherItems();
 		if (storePlayers) GatherPlayers();
 
-		SoundManager.PlayNetworkedAtPos("BinClose", DrawerWorldPosition, Random.Range(0.8f, 1.2f), sourceObj: gameObject);
+		// JESTER
+		// SoundManager.PlayNetworkedAtPos("BinClose", DrawerWorldPosition, Random.Range(0.8f, 1.2f), sourceObj: gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)